fuzzing/Main.hs

```diff
@@ -41,7 +41,7 @@ import Text.XML.Light (Attr(..), Element, ppTopElement, unode, unqual)
 
 -- | The @clang@ executable and flags for a particular test
 -- configuration, e.g., @("clang-3.8", "-O -w -g")@
```

Comment:
The comment should be updated  to match the type alias target.

Reply:
Ah, good point. I'll do that.